Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM memcached:latest
+
+USER root
+# install python for entrypoint
+RUN apt-get update \
+    && apt-get install -y \
+		netcat \
+    curl
+
+COPY containerbuddy/* /opt/containerbuddy/
```

Comment:
See pattern demonstrated in https://github.com/autopilotpattern/jenkins:

Containerbuddy config in `/etc` and helper scripts in `/bin` (which is copied to `/usr/local/bin`) or just `/usr/local/bin` outright.